keps/sig-api-machinery/2340-Consistent-reads-from-cache/README.md

```diff
@@ -51,16 +48,18 @@ Consistent reads may be served from cache so long as:
 - The data in the watch cache no older than the latest "revision" just from etcd
 
 etcd watches support "progress events", which provide an updated revision and a
-guarantee that all future watch events will be newer than the that revision.  If
-an etcd watch is configured with `WithProgressNotify` enabled, etcd
-automatically sends progress events at a regular interval. The "progress events"
-allow a etcd watcher to know how up-to-date the watch stream is relative a
-particular revision.
+guarantee that all future watch events will be newer than the that revision. 
+Etcd client can request a progress notification from server. The progress 
+notification allow the etcd watcher to know how up-to-date the watch stream 
+is. This is thanks to [bookmarkable] property of etcd watch that guarantees that
+all events with revision below progress notification have been delivered.
 
 This KEP summarizes how we can take advantage of progress events efficiently
 determine how up-to-date kubernetes watch caches are then serve reads from the
 watch cache when they are sufficiently up-to-date.
 
+[bookmarkable]: https://etcd.io/docs/v3.6/learning/api_guarantees/#watch-apis
+
```

Comment:
L84: in fact for this particular example of pods from a node, watchcache has a built-in index - might be worth mentioning

L107 (non-goals) - watchcache currently doesn't support pagination, so while we can serve the first page, continuation won't work.
Please it explicitly to non-goals that continuation will still be served from etcd.
[I want to eventually solve this problem, but this is a separate thing]

L110 - this unresolved comment has to be agreed on
I'm personally on the side that we don't have to expose the option to force the request to be served from etcd (which is how I understand David's comment), but I think @deads2k opinion is different here.

Reply:
Ypdated